unicodetools/src/main/java/org/unicode/tools/emoji/Emoji.java

```diff
@@ -101,51 +101,51 @@
-	    .put(VERSION13, UCD13)
-	    .put(VERSION12_1, UCD12_1)
-	    .put(VERSION12, UCD12)
-	    .put(VERSION11, UCD11)
-	    .put(VERSION5, UCD10)
-	    .put(VERSION4, UCD9)
-	    .put(VERSION3, UCD9)
-	    .put(VERSION2, UCD8)
-	    .put(VERSION1, UCD8)
-//	    .put(VERSION0_7, UCD7)
-//	    .put(VERSION0_6, UCD6)
-	    .build();
+        .put(VERSION14, UCD14)
+        .put(VERSION13_1, UCD13)
+        .put(VERSION13, UCD13)
+        .put(VERSION12_1, UCD12_1)
+        .put(VERSION12, UCD12)
+        .put(VERSION11, UCD11)
+        .put(VERSION5, UCD10)
+        .put(VERSION4, UCD9)
+        .put(VERSION3, UCD9)
+        .put(VERSION2, UCD8)
+        .put(VERSION1, UCD8)
+//        .put(VERSION0_7, UCD7)
+//        .put(VERSION0_6, UCD6)
+        .build();
 
     public final static Map<VersionInfo, String> EMOJI_TO_DATE = ImmutableMap.<VersionInfo, String>builder()
-	    .put(VERSION14, "2021-09-10")
-	    .put(VERSION13_1, "2020-09-10")
-	    .put(VERSION13, "2020-03-10")
-	    .put(VERSION12_1, "2019-10-29")
-	    .put(VERSION12, "2019-02-04")
-	    .put(VERSION11, "2018-02-07")
-	    .put(VERSION5, "2017-03-27")
-	    .put(VERSION4, "2016-11-22")
-	    .put(VERSION3, "2016-06-03")
-	    .put(VERSION2, "2015-11-12")
-	    .put(VERSION1, "2015-06-09")
-//	    .put(VERSION0_7, "2014-06-16")
-//	    .put(VERSION0_6, "2010-06-09")
-	    .build();
+        .put(VERSION14, "2021-09-10")
+        .put(VERSION13_1, "2020-09-10")
+        .put(VERSION13, "2020-03-10")
+        .put(VERSION12_1, "2019-10-29")
+        .put(VERSION12, "2019-02-04")
+        .put(VERSION11, "2018-02-07")
+        .put(VERSION5, "2017-03-27")
+        .put(VERSION4, "2016-11-22")
+        .put(VERSION3, "2016-06-03")
+        .put(VERSION2, "2015-11-12")
+        .put(VERSION1, "2015-06-09")
+//        .put(VERSION0_7, "2014-06-16")
+//        .put(VERSION0_6, "2010-06-09")
+        .build();
 
     public final static Map<Integer,VersionInfo> YEAR_TO_EMOJI_VERSION_ASCENDING;
     public final static Map<VersionInfo,Integer> EMOJI_VERSION_TO_YEAR;
     static {
-	Map<Integer,VersionInfo> _map = new TreeMap<>();
-	Map<VersionInfo,Integer> _mapEmojiToYear = new TreeMap<>();
-	for (Entry<VersionInfo, String> entry : EMOJI_TO_DATE.entrySet()) {
-	    int year = Integer.parseInt(entry.getValue().substring(0, 4));
-	    _mapEmojiToYear.put(entry.getKey(), year);
-	    if (!_map.containsKey(year)) {
-		_map.put(year, entry.getKey());
-	    }
-	}
-	YEAR_TO_EMOJI_VERSION_ASCENDING = ImmutableMap.copyOf(_map);
-	EMOJI_VERSION_TO_YEAR = ImmutableMap.copyOf(_mapEmojiToYear);
+    Map<Integer,VersionInfo> _map = new TreeMap<>();
+    Map<VersionInfo,Integer> _mapEmojiToYear = new TreeMap<>();
+    for (Entry<VersionInfo, String> entry : EMOJI_TO_DATE.entrySet()) {
+        int year = Integer.parseInt(entry.getValue().substring(0, 4));
+        _mapEmojiToYear.put(entry.getKey(), year);
+        if (!_map.containsKey(year)) {
+        _map.put(year, entry.getKey());
+        }
```

Comment:
This file does not look right. Appears to want 8sp/tab.

Reply:
Updated in https://github.com/unicode-org/unicodetools/pull/178/commits/aac5aa8e8c617d550104f1d3cd6ba6fd687950ff